engines/bastion_katello/app/assets/javascripts/bastion_katello/products/discovery/discovery.controller.js

```diff
@@ -50,6 +50,21 @@ angular.module('Bastion.products').controller('DiscoveryController',
             $scope.table.resource.subtotal = $scope.table.resource.total;
         };
 
+        $scope.filteredRows = function (filter) {
```

Comment:
This should really be an angular filter instead of a function in the controller but since all of the angular application is effectively deprecated I'm okay with it as long as it work.